### PR TITLE
Fixing server provisioning.

### PR DESCRIPTION
**Pull request by chngtrn**

### CODE REVIEW

**Comment by tbigby-kristin** [comment]
I just tested this when rebuilding my cluster - the wait based on netstat continued, then the wait for loginresponse worked well: the first three times it tried there was no response, on the fourth time the response came through and then the script completed.

**Comment by saamalik** [comment]
This worked for me too! Please merge.

File: vagrant/scripts/configure_rancher_server.sh [hunk 1]
```diff
@@ -27,15 +27,18 @@ $rancher_command
 
 # wait until rancher server is ready
 while true; do
-  wget -T 5 -c https://localhost/ping && break
```
Comment (rolandg) [inline vagrant/scripts/configure_rancher_server.sh]:
```
wget -T 5 -c http://localhost/ping && break
```
is also working

**Comment by superseb** [comment]
Thanks, this is now corrected

**Pull request closed without merging**